generator/types.go

```diff
@@ -312,6 +324,26 @@ func (m Method) HasParams() bool {
 	return len(m.Params) > 0
 }
 
+// HasContext returns true if method has a context
+func (m Method) HasContext() bool {
+	for _, p := range m.Params {
+		if p.Type == "context.Context" {
+			return true
+		}
+	}
+	return false
+}
+
+// GetContext returns the variable name of the context param
+func (m Method) GetContext() string {
+	for _, p := range m.Params {
+		if p.Type == "context.Context" {
+			return p.Name
+		}
+	}
+	return ""
+}
+
```

Comment:
Gowrap already has this:
* PTAL at the AcceptsContext method helper
* If method accepts context the context argument is always called "ctx"